src/app/events/[eventId]/edit/page.tsx

```diff
@@ -1,11 +1,10 @@
 'use client'
 
-export default function edit() {
+export default function edit( { params }: { params: { eventId:string } }) {
     const makeApiCall = async () => {
 
-        await fetch('/api', {
+        await fetch(`/api/events/${params.eventId}`, {
             method: 'GET',
```

Comment:
Function is called edit, but the method is a `GET`?

Reply:
This will be changed. I was misusing that endpoint just for testing. I will add a comment.